transfer-archive/app/helpers/settings.py

```diff
@@ -0,0 +1,29 @@
+from pydantic import BaseSettings, validator, SecretStr
+
+try:
+    from dotenv import load_dotenv
+    load_dotenv()
+except ModuleNotFoundError:
+    pass
+
+
+class Settings(BaseSettings):
+    """
+    Environment variables
+    """
+    iam_access_key_id: str
+    iam_secret_access_key: SecretStr
+    datasett_id: str
+    s3_endpoint_url: str
+    bucket_name: str
+    s3_path: str
+    azure_container: str
+    azure_base_url: str
+    sas_generator_host: str
+    rclone_config_path: str = "/root/.config/rclone/rclone.conf"
+
+    @validator('*')
+    def not_empty(cls, value):
+        if not value:
+            raise ValueError("Environment variable can not be empty")
```

Comment:
```suggestion
        assert value != '', "Environment variable can not be empty"
```

Nå er ingen av env variablene booleans, men denne valideringen ville slått ut f.eks dersom `s3_path` hadde type `bool` og var satt til `False`.
Så er bedre å `assert value != ''`, fordi da sjekker du kun om det er en empty string, og ikke om det kan være en False boolean